src/Symfony/Component/Form/FormError.php

```diff
@@ -185,6 +185,10 @@ public function serialize()
      */
     public function unserialize($serialized)
     {
-        list($this->message, $this->messageTemplate, $this->messageParameters, $this->messagePluralization, $this->cause) = unserialize($serialized);
+        if (PHP_VERSION_ID >= 70000) {
+            list($this->message, $this->messageTemplate, $this->messageParameters, $this->messagePluralization, $this->cause) = unserialize($serialized, array('allowed_classes' => false));
```

Comment:
this looks wrong to me. The cause can be an object (and actually, it *is* an object in many cases in Symfony)